src/tutorial/index.rst

```diff
@@ -541,10 +533,52 @@ The remainder of the Circles model's message input agent function contains some
     }
     """
     ...
+
+  .. code-tab:: py Agent Python
+    
+    ...
+    # Agent Function to read the location messages and decide how the agent should move
+    @pyflamegpu.agent_function
+    def input_message(message_in: pyflamegpu.MessageSpatial2D, message_out: pyflamegpu.MessageNone):
+        ID = pyflamegpu.getID()
+        REPULSE_FACTOR = pyflamegpu.environment.getPropertyFloat("repulse")
+        RADIUS = message_in.radius()
+        fx = 0.0
+        fy = 0.0
+        x1 = pyflamegpu.getVariableFloat("x")
+        y1 = pyflamegpu.getVariableFloat("y")
+        count = 0
+        for (message : message_in(x1, y1)) :
```

Comment:
that's not a python style loop?

Reply:
There were a few code typos I found when putting all this in to the full example. I thought I had ported them all back into the various places but must have missed this one. Now fixed.